sycl-fusion/jit-compiler/lib/fusion/FusionPipeline.h

```diff
@@ -27,6 +27,13 @@ class FusionPipeline {
   static std::unique_ptr<SYCLModuleInfo>
   runFusionPasses(llvm::Module &Mod, SYCLModuleInfo &InputInfo,
                   BarrierFlags BarriersFlags);
+
+  ///
+  /// Run the necessary passes in a custom pass pipeline to perform
+  /// materialization of kernel specialization constants.
+  static bool runMaterializerPasses(llvm::Module &Mod,
+                                    const unsigned char *SpecConstData,
+                                    size_t SpecConstDataSize);
```

Comment:
```suggestion
                                    ArrayRef<unsigned char> SpecConst);
```

Keep coupled here and in the rest of the code

Reply:
Wooft, `View` provides a `to<T>` conversion, that is neat!